udata/templates/mail/account_deleted.txt

```diff
@@ -0,0 +1,6 @@
+{% extends theme('mail/base.txt') %}
+
+{% block body %}
+{{ _('Your account has now been deleted') }}.
```

Comment:
We don't recap in this story because datasets and organizations (if there is some), are not deleted. Right now it's already allowed.
We only delete the account. Another story might prompt the admin for what to do with the related content (more like a moderation)